src/components/ImageWithSizeCalculation.js

```diff
@@ -16,11 +20,25 @@ const propTypes = {
 
     /** Callback fired when the image has been measured. */
     onMeasure: PropTypes.func,
+
+    /** Does the image require an authToken? */
+    isAuthTokenRequired: PropTypes.bool,
+
+    /* Onyx props */
+    /** Session object */
+    session: PropTypes.shape({
+        /** An error message to display to the user */
+        encryptedAuthToken: PropTypes.string,
+    }),
```

Comment:
Why was this added - it's not used in the `ImageWithSizeCalculation` component? 
Perhaps it's a leftover for something else you tried?